lib/Conversion/XTenToLinalg.cpp

```diff
@@ -202,7 +202,8 @@ class XTenMMOpConversion : public ConversionPattern {
     auto oper0Ty = operands[0].getType().cast<Torch::BaseTensorType>();
     auto oper1Ty = operands[1].getType().cast<Torch::BaseTensorType>();
     auto dtype = tTy.getDtype();
-    std::vector<int64_t> sizes{oper0Ty.getSizes()[1], oper1Ty.getSizes()[0]};
+    std::vector<int64_t> sizes{oper0Ty.getSizes()[0], oper1Ty.getSizes()[1]};
```

Comment:
And this didn't trigger the need to update any test 😢

Reply:
Thanks! Forgot about adding a test.